app/views/groups/index.html.erb

```diff
@@ -47,6 +47,10 @@
   <%= raw t('groups.index.instructions', {
       icon: '<i class="fa fa-plus-circle"></i>'
     }) %>
+  <br /><br />
+  <h1 class="faq_heading"><%= t('pages.faq.group_question') %></h1>
+  <br />
```

Comment:
You shouldn't need to put a `<br>` here since there's a `<h1>` above it.

Reply:
yes, one wouldn't think so. :) But when I tested it, there was no line break. Strange, isn't it. I'll try it your way again, and test it when the patch is done. Any problems and I'll follow up with you on that.